pkgs/applications/networking/instant-messengers/simplex-chat/simplex-chat-desktop.nix

```
@@ -0,0 +1,46 @@
+{ lib
+  , appimageTools
+  , fetchurl
+}:
+
+let
+  pname = "simplex-chat-desktop";
+  version = "5.3.0";
+
+  src = fetchurl {
+    url = "https://github.com/simplex-chat/simplex-chat/releases/download/v${version}/simplex-desktop-x86_64.AppImage";
+    hash = "sha256-inGg6KbHO1BZnmgr96AQyi6gOds/Vve3CGR02PnfCwc=";
+  };
+
+  appimageContents = appimageTools.extract {
+    inherit pname version src;
+  };
+in appimageTools.wrapType2 {
+    inherit pname version src;
+
+    extraPkgs = pkgs: with pkgs; [
+      makeWrapper
+    ];
+
+    extraBwrapArgs = [
+      "--setenv _JAVA_AWT_WM_NONREPARENTING 1"
+    ];
+
+    extraInstallCommands = ''
+      mv $out/bin/${pname}-${version} $out/bin/${pname}
+
+      install --mode=444 -D ${appimageContents}/chat.simplex.app.desktop --target-directory=$out/share/applications
+      substituteInPlace $out/share/applications/chat.simplex.app.desktop \
+        --replace 'Exec=AppRun' 'Exec=${pname}'
+      cp -r ${appimageContents}/usr/share/icons $out/share
+    '';
+
+  meta = with lib; {
+    description = "Desktop application for SimpleX Chat";
+    homepage = "https://simplex.chat";
+    changelog = "https://github.com/simplex-chat/simplex-chat/releases/tag/v${version}";
+    license = licenses.agpl3Only;
+    maintainers = with maintainers; [ yuu ];
+    platforms = with platforms; [ "x86_64-linux" ];
```

`lib.platforms` isn't needed here

```suggestion
    platforms = [ "x86_64-linux" ];
```

---

done that but what is going on with the `simplex-chat-desktop, simplex-chat-desktop.passthru.tests on aarch64-darwin` that didn't skip? https://logs.ofborg.org/?attempt_id=e105b7db-ecda-46fc-b43a-3167e2899e77&key=nixos%2Fnixpkgs.256615